python/tests/integration/planning/sequential_planner/test_sequential_planner.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+
+import semantic_kernel.connectors.ai.open_ai as sk_oai
+from semantic_kernel.kernel import Kernel
+from semantic_kernel.planning.sequential_planner.config import SequentialPlannerConfig
+from semantic_kernel.planning.sequential_planner.planner import SequentialPlanner
+from tests.integration.fakes.email_skill_fake import EmailSkillFake
+from tests.integration.fakes.fun_skill_fake import FunSkillFake
+from tests.integration.fakes.writer_skill_fake import WriterSkillFake
+
+
+def initialize_kernel(get_aoai_config, use_embeddings=False, use_chat_model=False):
+    deployment_name, api_key, endpoint = get_aoai_config
+
+    kernel = Kernel()
+    if use_chat_model:
+        kernel.add_chat_service(
+            "chat_completion",
+            sk_oai.AzureChatCompletion(deployment_name, endpoint, api_key),
+        )
+    else:
+        kernel.add_text_completion_service(
+            "text_completion",
+            sk_oai.AzureChatCompletion(deployment_name, endpoint, api_key),
+        )
+
+    if use_embeddings:
+        kernel.add_text_embedding_generation_service(
+            "text_embedding",
+            sk_oai.AzureTextEmbedding("text-embedding-ada-002", endpoint, api_key),
+        )
+    return kernel
+
+
+@pytest.mark.parametrize(
+    "use_chat_model, prompt, expected_function, expected_skill",
+    [
+        (
+            False,
+            "Write a joke and send it in an e-mail to Kai.",
+            "SendEmail",
+            "_GLOBAL_FUNCTIONS_",
+        ),
+        (
+            True,
+            "Write a joke and send it in an e-mail to Kai.",
+            "SendEmail",
+            "_GLOBAL_FUNCTIONS_",
+        ),
+    ],
+)
+@pytest.mark.asyncio
+async def test_create_plan_function_flow_async(
+    get_aoai_config, use_chat_model, prompt, expected_function, expected_skill
+):
+    # Arrange
+    kernel = initialize_kernel(get_aoai_config, False, use_chat_model)
+    kernel.import_skill(EmailSkillFake())
+    kernel.import_skill(FunSkillFake())
+
+    planner = SequentialPlanner(kernel)
+
+    # Act
+    plan = await planner.create_plan_async(prompt)
+
+    # Assert
+    assert any(
+        step.name == expected_function and step.skill_name == expected_skill
+        for step in plan._steps
+    )
+
+
+@pytest.mark.parametrize(
```

Comment:
This test in particular is quite flaky. Sometimes it passes, but more often not. Trying to figure out why.

Reply:
I've run this test many times and it always passes 😭  
Could you show me an error log for that failure?